appshell/client_handler_mac.mm

```diff
@@ -56,9 +56,12 @@
   NSWindow* window = [view window];
   std::string titleStr(title);
   NSString* str = [NSString stringWithUTF8String:titleStr.c_str()];
-  [window setTitle:str];
```

Comment:
message in XCode console -
2016-05-12 23:14:31.524 Brackets Helper[2782:100768] Couldn't set selectedTextBackgroundColor from default ()